instructionAPI/src/Instruction.C

```diff
@@ -92,6 +92,8 @@ namespace Dyninst { namespace InstructionAPI {
           return id != amdgpu_gfx90a_op_INVALID;
         case Arch_amdgpu_gfx940:
           return id != amdgpu_gfx940_op_INVALID;
+        case Arch_riscv64:
```

Comment:
Could you add a blank line above this to visually separate it from the amdgpu ones?

Reply:
Fixed